### PR TITLE
 Documentation for EmployeeTrainingPrograms model

### PR DESCRIPTION
**Pull request by lsrichert**
Purpose:
Added documentation for author and purpose of the EmployeeTrainingProgram model

How it fits in context of the project:
Clarifies what this model contains at the top of the file

Specific Feature affected:
EmployeeTrainingProgram model contains commented out documentation; no code affected

How to test (Be thorough!):

1. check into branch LR-EmpTrainModelDocumentation

2. open EmployeeTrainingPrograms.cs

3. make sure the documentation at the top is there and commented out

### CODE REVIEW

**Review by Rian501 — COMMENTED** [review]
Lauren, you are using bracket things like [Required], [Key], etc. Please explain what these mean. Or, take them out.

**Comment by lsrichert** [comment]
[Key] is an attribute that identifies the primary key. I believe that if you don't use this attribute in your code, the primary key will be identified as 'id' rather than 'key'. So using the attribute is not necessary but I have left it in my code.
[Required] Indicates that the property is a required field. By using this attribute, a 'NOT NULL' column will be created for any property on which it is applied. I don't think this is really necessary either because you can specify a property as 'NOT NULL' in the query when creating the table. but I have left this in my code as well.

**Comment by lsrichert** [comment]
I have removed the attributes [Key] and [Required]

File: Models/Products.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿using BangazonAPI.Models;
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
```
Comment (AmandaM04) [inline Models/Products.cs:4]: needs to be deleted

**Review by JewelRam — APPROVED** [review]

File: Models/Products.cs [hunk 2]
```diff
@@ -13,8 +14,8 @@ public class Products
         
 
         [Required]
```
Comment (AmandaM04) [inline Models/Products.cs:16]: needs to be deleted

File: Models/Products.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿using BangazonAPI.Models;
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
```
Comment (AmandaM04) [inline Models/Products.cs:5]: needs to be deleted

File: Models/Products.cs (hunk 1 above)
Comment (lsrichert) [reply Models/Products.cs:4]: I had to fix a small error in Products for my controller to work. Brett is working to remove the attributes in the models he created.

File: Models/Products.cs (hunk 3 above)
Comment (lsrichert) [reply Models/Products.cs:5]: Brett is working on this deletion

File: Models/Products.cs (hunk 2 above)
Comment (lsrichert) [reply Models/Products.cs:16]: Brett is working on this deletion from Products.

**Pull request closed without merging**